common/transport/amqp/src/amqp_message.ts

```diff
@@ -3,14 +3,22 @@
 
 'use strict';
 
+import { uuid_to_string as rheaUuidToString, string_to_uuid as rheaStringToUuid, types as rheaTypes, message as rheaMessage } from 'rhea';
```

Comment:
nit: couldn't you do import * as rhea from 'rhea' then used reah.uuid_to_string, etc, instead of redefining each individual name?

Reply:
additional comment: rheaMessage is not accurate.  maybe call this messageUtils or something similar?

---
In reply to: [209365550](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209365550) [](ancestors = 209365550)